web/webplatform/webplatform_test.go

```diff
@@ -27,3 +28,20 @@ func TestLoad(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, f)
 }
+
+func TestVersion(t *testing.T) {
+	l, _ := zap.NewDevelopment() // if test fails, will show detailed version error.
```

Comment:
Check for error, otherwise test failures will be hard to understand.

Reply:
removed test